src/Console/Commands/ExportSpecifications.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Dedoc\Scramble\Console\Commands;
+
+use Dedoc\Scramble\Generator;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\File;
+
+class ExportSpecifications extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'scramble:export
+        {--path= : The path where to save the exported json file}
+    ';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Export the OpenAPI specifications to a json file.';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(Generator $generator): void
+    {
+
+        $specifications = json_encode($generator());
+
+        /** @var string filename */
+        $filename = $this->hasOption('path')
```

Comment:
@princejohnsantillan I am trying this and it seems like `$this->hasOption('path')` is always returning `true` which causes the command to fail when a path isn't provided, I would suggest adding a test to cover this as your only test covers passing with the path manually

Reply:
Thanks @logicaldox. Apparently `hasOption` is not the check I thought it was. It's checking the signature not the input. Your suggestion works perfectly and I applied it. Thanks again!